ui/pages/onboarding-flow/import-srp/import-srp.js

```diff
@@ -57,6 +57,17 @@ export default function ImportSRP({ submitSecretRecoveryPhrase }) {
   }, [currentKeyring, history]);
   const trackEvent = useContext(MetaMetricsContext);
 
+  const onShowSrpDetailsModal = useCallback(() => {
+    trackEvent({
+      category: MetaMetricsEventCategory.Onboarding,
+      event: MetaMetricsEventName.SrpDefinitionClicked,
+      properties: {
+        location: 'import_srp',
```

Comment:
maybe all locations should also be constants. ideally, the route name.
sync with segment schema